### PR TITLE
Adds support for pcb in the hardware process

### PR DESCRIPTION
**Pull request by Bucknalla**
Change-type: major
Signed-off-by: Alex Bucknall <alexbucknall@balena.io>

### CODE REVIEW

**Comment by Bucknalla** [comment]
@anujdeshpande I realise there's another PR with a similar thing in place but I'd rather work to craft a minimum viable RFQ generator for now and then add what we need, as we need it

File: hardware-types/pcb/fileTypes.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "fileTypes" : ["gbr", "pdf", "drl", "txt", "csv"]
```
Comment (anujdeshpande) [inline hardware-types/pcb/fileTypes.json]: @Bucknalla should we add xlsx, zip here as well?
Plus SchDoc, and other Altium ones?

Comment (Bucknalla) [reply hardware-types/pcb/fileTypes.json]: @anujdeshpande did we get a comprehensive list from the EtcherPro guys?

Comment (anujdeshpande) [reply hardware-types/pcb/fileTypes.json]: I see the following in [EP's drive folder](https://drive.google.com/drive/u/0/folders/1Ps1dsA8fveKnyBnzIjDWWAXcwzFug-a7). I think it's safe to include atleast these and then see what breaks @Bucknalla 
- xlsx
- zip (3D and gerbers)
- SchDoc
- BomDoc
- PcbDoc
- PrjPcbStructure
- OutJob

Comment (anujdeshpande) [reply hardware-types/pcb/fileTypes.json]: cc @zisis00 @ntzovanis

Comment (ntzovanis) [reply hardware-types/pcb/fileTypes.json]: @anujdeshpande not sure what you need here. Is it design files extensions?

Comment (anujdeshpande) [reply hardware-types/pcb/fileTypes.json]: @ntzovanis yepp sorry for not giving more context - but this will help build RFQs for PCB projects

Comment (zisis00) [reply hardware-types/pcb/fileTypes.json]: @anujdeshpande Please add also the “.PrjPcb”. For the “.PrjPcbStructure” is not necessary as it is created when you open the “.PrjPcb”.

**Review by anujdeshpande — APPROVED** [review]

**Pull request merged**